# Identify the quadrant (when applicable)

uint8 NO_QUADRANT = 0
uint8 QUADRANT_1  = 1
uint8 QUADRANT_2  = 2
uint8 QUADRANT_3  = 3
uint8 QUADRANT_4  = 4

uint8 data
